case-lib/hijack.sh

```diff
@@ -2,8 +2,9 @@
 
 SUDO_CMD=$(command -v sudo)
 
+trap 'func_hijack_exit' EXIT
 # Overwrite other functions' exit to perform environment cleanup
-function exit()
+function func_hijack_exit()
```

Comment:
You can remove "hijack" from the name, this is not hijacking anything any more. Suggestions: `cleanup_on_exit()` or `exit_handler()`.